tests/commands/test_pipelines_commands.py

```diff
@@ -229,6 +229,41 @@ def test_get_info_api_exception_no_user_found(capture_logs):
     )
 
 
+def test_get_info_api_exception_401_unauthorized(capture_logs):
+    runner = CliRunner()
+    pipeline_name = "whatever"
+
+    when(pipelines_commands.pipelines_logic).get_pipeline_info(
+        pipeline_name, None
+    ).thenRaise(
+        ApiException(
+            status=401,
+            reason="Unauthorized",
+            body="""<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html><head>
```

Comment:
this spacing annoys me but i think i can let it pass 😂

Reply:
better now?

Reply:
🥂